man/man8/zpool-resilver.8

```diff
@@ -27,29 +26,27 @@
 .\" Copyright 2017 Nexenta Systems, Inc.
 .\" Copyright (c) 2017 Open-E, Inc. All Rights Reserved.
 .\"
-.Dd August 9, 2019
+.Dd May 27, 2021
 .Dt ZPOOL-RESILVER 8
 .Os
+.
 .Sh NAME
 .Nm zpool-resilver
-.Nd Start a resilver of a device in a ZFS storage pool
+.Nd resilver devices in ZFS storage pool
```

Comment:
nit: keep the "a" for the pool (not the resilver)

Reply:
as above